packages/manager/modules/office/src/dashboard/microsoft-office-license.service.js

```diff
@@ -35,8 +45,10 @@ export default class MicrosoftOfficeLicenseService {
    * @return {[string]}
    */
   get(licenseId) {
-    return this.$http
-      .get(`${this.basePath}/${licenseId}`)
+    const promise = MicrosoftOfficeLicenseService.isPrepaidService(licenseId)
+      ? this.licensePrepaidService.get(licenseId)
+      : this.licensePostpaidService.get(licenseId);
+    return promise
       .then((response) => response.data)
       .catch((err) => this.$q.reject(err.data));
```

Comment:
May we could do that:

```suggestion
    const isPrepaid = MicrosoftOfficeLicenseService.isPrepaidService(licenseId);
    const service = isPrepaid ? 'licensePrepaidService' : 'licensePostpaidService';
    
    return this[service].get(licenseId)
      .then((response) => response.data)
      .catch((err) => this.$q.reject(err.data));
```

Reply:
Initially this was my approach but we can not apply this method for all the functions since the arguments will vary for many functions. To maintain the uniformity for all the functions, took this common approach. Very good suggestion though.

Reply:
For uniformity raisons, ok got it.